content/en/docs/languages/go/generated-code.md

```diff
@@ -34,68 +34,54 @@ In this context, `MsgA` is the protobuf message sent from the client, and `MsgB`
 ### Server-streaming methods
 
 These methods have the following signature on the generated service interface:
+`Foo(*MsgA, grpc.ServerStreamingServer[*MsgB]) error`
 
-`Foo(*MsgA, <ServiceName>_FooServer) error`
+In this context, `MsgA` is the single request from the client, and `grpc.ServerStreamingServer` represents the server side of server-to-client stream of response type `MsgB`.
 
-In this context, `MsgA` is the single request from the client, and the `<ServiceName>_FooServer` parameter represents the server-to-client stream
-of `MsgB` messages.
-
-`<ServiceName>_FooServer` has an embedded `grpc.ServerStream` and the following interface:
+`<ServiceName>_FooServer` has a `grpc.ServerStreamingServer` interface that takes `MsgB`, i.e. the response type, as the input.
```

Comment:
I think we shouldn't mention `<ServiceName>_FooServer`, since that name is no longer part of the generated code, except for backward compatibility.

But I wonder if we should have two documents, and link between them: one for the old generated code and one for the new generated code.  So we can keep the old code for people that are working with an old generated file.

Reply:
Done.